decidim-elections/app/views/decidim/elections/admin/steps/_vote.html.erb

```diff
@@ -13,3 +17,5 @@
     <% end %>
   </div>
 </div>
+
+<%= javascript_include_tag("decidim/elections/admin/vote_statistics") %>
```

Comment:
You'll need to add the file to the precompile list, otherwise this will fail in production. Here's an example:

https://github.com/decidim/decidim/blob/804be88918895e63da14cc899f4e787f2eeefe62/decidim-forms/lib/decidim/forms/admin_engine.rb#L12-L14

Reply:
Thanks a lot. I'll add it! 😊